admin/src/pages/UserSearch.vue

```diff
@@ -41,9 +42,27 @@ export default {
         { key: 'lastName', label: 'Lastname' },
         {
           key: 'creation',
-          label: 'Creation',
+          label: 'open Creation',
```

Comment:
Why do we write it 'open Creation' and not 'Open creations'
```suggestion
          label: 'Open creations',
```